mlflow/pipelines/steps/train.py

```diff
@@ -457,6 +465,43 @@ def render_schema(inputs, title):
         else:
             run_card_tab.add_markdown("MODEL_URI", f"**MLflow Model URI:** `{model_uri}`")
 
+        # Tab 8: Best Parameters
+        if self.step_config["tuning"]["enabled"]:
```

Comment:
Instead of check the step conf here, can we have the `_build_step_card` method accept a `best_parameters` object and populate tabs depending on whether this object is present? Ditto for `hp_trials_df` below